com.unity.render-pipelines.universal/Runtime/Passes/PostProcessPass.cs

```diff
@@ -226,45 +231,69 @@ RenderTextureDescriptor GetStereoCompatibleDescriptor(int width, int height, Gra
 
         bool RequireSRGBConversionBlitToBackBuffer(CameraData cameraData)
         {
-            bool requiresSRGBConversion = Display.main.requiresSrgbBlitToBackbuffer;
-            // For stereo case, eye texture always want color data in sRGB space.
-            // If eye texture color format is linear, we do explicit sRGB convertion
-#if ENABLE_VR && ENABLE_VR_MODULE
-            if (cameraData.isStereoEnabled)
-                requiresSRGBConversion = !XRGraphics.eyeTextureDesc.sRGB;
-#endif
-            return requiresSRGBConversion;
+            return cameraData.requireSrgbConversion && m_EnableSRGBConversionIfNeeded;
+        }
+
+        // TODO: deprecate this version in favor of the above with load/store actions
+        private new void Blit(CommandBuffer cmd, RenderTargetIdentifier source, RenderTargetIdentifier destination, Material material, int passIndex = 0)
+        {
+            cmd.SetGlobalTexture(ShaderPropertyId.sourceTex, source);
+            if (m_UseDrawProcedural)
+            {
+                Vector4 scaleBias = new Vector4(1, 1, 0, 0);
+                cmd.SetGlobalVector(ShaderPropertyId.scaleBias, scaleBias);
+
+                cmd.SetRenderTarget(new RenderTargetIdentifier(destination, 0, CubemapFace.Unknown, -1),
+                    RenderBufferLoadAction.Load, RenderBufferStoreAction.Store, RenderBufferLoadAction.Load, RenderBufferStoreAction.Store);
+                cmd.DrawProcedural(Matrix4x4.identity, material, passIndex, MeshTopology.Quads, 4, 1, null);
+            }
```

Comment:
1. This seems like a duplication of RenderUtils.Blit(), can you combine them?
2. If this is a replacement for native command buffer Blit, it should be on parity with native functions (such as support for HDR buffers, etc).